src/annotator/events.ts

```diff
@@ -0,0 +1,16 @@
+import type { SidebarLayout } from '../types/annotator';
+
+type LayoutChangeEventDetail = {
+  isSideBySideActive: boolean;
```

Comment:
`sideBySideActive` would be a more consistent name with eg .`sidebarLayout.expanded` and other DOM properties.